drv.go

```diff
@@ -356,6 +356,13 @@ func (d *drv) acquireConn(pool *connPool, P commonAndConnParams) (*C.dpiConn, bo
 				cs := C.CString(value)
 				defer C.free(unsafe.Pointer(cs))
 				C.dpiData_setBytes(&tempData, cs, C.uint32_t(len(value)))
+			case []byte:
+				columns[i].oracleTypeNum = C.DPI_ORACLE_TYPE_RAW
+				columns[i].nativeTypeNum = C.DPI_NATIVE_TYPE_BYTES
+				bytesVal := value.([]byte)
```

Comment:
value is already []byte, you don't need the conversion and bytesVal.

Reply:
True. Will change this.